### ISSUE TITLE
pool of coupons for JPAL staff

### ISSUE THREAD

**Issue by pdpinch** [mitodl/mitxonline#1327]
As a benefit to JPAL staff, they are allowed to take DEDP courses for free. 

We need a pool of 25 coupons for each of the courses in the DEDP program that the DEDP team will distribute to staff on request. 

#### Acceptance Criteria:

- [ ] A google sheet of 7 x 25 coupons (I don't care if it's one tab or seven, as long as it's clear which course each coupon belongs too)

**Comment by jkachel** [comment]
I've created these codes and put them in a sheet called "MIT JPAL Staff Discounts" in Google Drive in the same place as last time. I've used `JPAL-course-` as the prefix (with some liberties taken with the course ID; we get a max of 13 characters there). The codes are all one-time use and are fixed-price set to $0. 

One thing to note - I haven't assigned products to these as products are per course run. I can do that but it would mean the codes would stop being usable after the course run has finished (so, after 1T2023).
